data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ( 'Agumon', '2020-02-03', 0, false, 10.23 );
+
```

Comment:
one of the values you inputted is incorrect please check it and try again 😸

Reply:
Noted and fixed 💯